app/utils/mycrypto.ts

```diff
@@ -0,0 +1,12 @@
+/*
+   Doc https://support.mycrypto.com/how-to/sending/creating-prefilled-transaction
+   at moment when code was prepared 25.11.2020 beta functionality didn't work
+ */
+
+export const myCryptoUrl = (
```

Comment:
So there is no gasLimit from myCrypto?

Reply:
Sadly not but they have option (by default turned on) to automatically calculate it